src/contexts/NetworkContext.js

```diff
@@ -0,0 +1,61 @@
+import React, { createContext, useState, useEffect, useContext } from 'react';
+import storage from '../services/storage';
+
+const NetworkConext = createContext();
+
+/**
+ * Custom hook to subscribe to network status.
+ *
+ * Value is retrieved from local storage initially and updated by service worker firing events
+ *
+ * Can be used by any functional component under the NetworkContextProvider in App.js
+ *
+ * @returns {boolean} true if connected to the network, false otherwise.
+ */
+export const useNetworkStatus = () => {
+  const context = useContext(NetworkConext);
```

Comment:
Is this missing a 't'?

Reply:
I did indeed misspell Context when creating the variable 🤦. Thanks for catching that! I've pushed a fix.